oidcclient/jwk.go

```diff
@@ -21,19 +21,27 @@ import (
 )
 
 const defaultJwkExpiration = 15 * time.Minute
-const zoneIDHeader = "x-zone_uuid"
+const appTIDHeader = "x-app_tid"
+const clientIDHeader = "x-client_id"
+const azpHeader = "x-azp"
 
-// OIDCTenant represents one IAS tenant correlating with one zone with it's OIDC discovery results and cached JWKs
+// OIDCTenant represents one IAS tenant correlating with one app_tid and client_id with it's OIDC discovery results and cached JWKs
 type OIDCTenant struct {
 	ProviderJSON    ProviderJSON
-	acceptedZoneIds map[string]bool
+	acceptedTenants map[TenantInfo]bool
```

Comment:
Can we rename this to `acceptedAppTenants` or `acceptedClients`? 
I was confused because I was thinking of IAS tenants.